app/test/task/task_test.dart

```diff
@@ -675,6 +678,59 @@ void main() {
           TestUser(email: 'admin@pub.dev', likes: []),
         ],
       ));
+
+  group('latest analyzed version', () {
```

Comment:
I'd be fine splitting this file in two.

Maybe we just put all the fake_time testing `task_faketime_test.dart` ?